src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -308,6 +312,18 @@ protected ImagePullPolicy deduceImagePullPolicy(AppDeploymentRequest request) {
 		return requests;
 	}
 
+	/**
+	 * Get the hostNetwork setting for the deployment request.
+	 *
+	 * @param request The deployment request.
+	 * @return Whether host networking is requested
+	 */
+	protected boolean getHostNetwork(AppDeploymentRequest request) {
+		String hostNetwork =
+				request.getDeploymentProperties().get("spring.cloud.deployer.kubernetes.hostNetwork");
```

Comment:
Currently, the property `spring.cloud.deployer.kubernetes.hostNetwork` is not part of `KubernetesDeployerProperties`. Is there a reason for it? I guess having this property there would be useful to set at the startup/app level vs deployment property only.